www/src/components/homepage/homepage-logo-banner.js

```diff
@@ -2,66 +2,52 @@
 import { jsx } from "theme-ui"
 import { useStaticQuery, graphql } from "gatsby"
 import Img from "gatsby-image"
-import styled from "@emotion/styled"
-import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
 import { Name } from "./homepage-section"
 
-const Section = styled(`section`)`
-  border-bottom: 1px solid ${p => p.theme.colors.ui.border};
-  overflow: hidden;
-  padding: ${p => p.theme.space[5]} 0;
-  width: 100%;
+const Section = props => (
+  <section
+    {...props}
+    sx={{
+      borderBottom: t => `1px solid ${t.colors.ui.border}`,
+      overflow: `hidden`,
+      py: [5, null, null, null, null, null, 7],
+      my: [null, null, null, null, null, `-1px`],
+      width: `100%`,
+    }}
+  />
+)
 
-  ${mediaQueries.xl} {
-    margin-top: -1px;
-    margin-bottom: -1px;
-  }
+const Title = props => (
+  <header
+    {...props}
+    sx={{
+      px: [6, null, null, null, null, `5%`, `8%`],
+      maxWidth: [null, null, null, `30rem`],
+      ml: [null, null, null, null, 9],
+    }}
+  />
+)
 
-  ${mediaQueries.xxl} {
-    padding: ${p => p.theme.space[7]} 0;
-  }
-`
-
-const Title = styled(`header`)`
-  padding-right: ${p => p.theme.space[6]};
-  padding-left: ${p => p.theme.space[6]};
-  ${mediaQueries.md} {
-    max-width: 30rem;
-  }
-
-  ${mediaQueries.lg} {
-    margin-left: ${p => p.theme.space[9]};
-  }
-
-  ${mediaQueries.xl} {
-    padding-right: 5%;
-    padding-left: 5%;
-  }
-
-  ${mediaQueries.xxl} {
-    padding-right: 8%;
-    padding-left: 8%;
-  }
-`
-
-const LogoGroup = styled(`div`)`
-  position: relative;
-  display: grid;
-  grid-auto-flow: column;
-  grid-auto-columns: auto;
-  grid-gap: ${p => p.theme.space[8]};
-  align-items: center;
-  overflow-x: scroll;
-  padding-left: ${p => p.theme.space[3]};
-  padding-bottom: ${p => p.theme.space[4]};
-  ${mediaQueries.xxl} {
-    padding-bottom: ${p => p.theme.space[6]};
-  }
-  &::-webkit-scrollbar {
-    display: none;
-  }
-`
+const LogoGroup = props => (
```

Comment:
Btw. I feel like up to this object length I'd go inline without even thinking about it.
So … what are your thoughts re:"responsive arrays" @tesseralis?

Reply:
Could you leave more details on this? I'm not sure what "responsive arrays" are

Reply:
`[null, null, null, "20px"]` — https://theme-ui.com/sx-prop/#responsive-values

Reply:
Ohhhhhhhh. Yeah they're great :P

Reply:
For me there are cases where I prefer to not use them, … and also the more `null`s, the more torn I am :)

Reply:
Btw. did you see the v1 roadmap?
https://github.com/system-ui/theme-ui/issues/832